wiring/inc/spark_wiring_usartserial.h

```diff
@@ -31,6 +31,16 @@
 #include "usart_hal.h"
 #include "spark_wiring_platform.h"
 
+//Available Serial Configuration for C
```

Comment:
Could you please move these constants down into the HAL layer, into `usart_hal.h` since they are part of the HAL interface. At present, the new code in the HAL contains implicit magic constants for the usart config - having these available to the HAL will mean we can use more meaningful symbols instead.